nitrogen: add page (#7492)
tldr-pages/tldr: pages/linux/nitrogen.md

# nitrogen

> > Nitrogen is a fast and lightweight desktop background browser and setter for X Window.

Reviewer: ```suggestion
> Desktop background browser and setter for X Window.
```

> > More information: <https://github.com/l3ib/nitrogen/>.

Reviewer: ```suggestion
> More information: <https://github.com/l3ib/nitrogen>.
```

> - View and set the desired wallpaper/s from a specific directory:

Reviewer: The `/` only adds noise
```suggestion
- View and set the wallpapers from a specific directory:
```

> `nitrogen {{path/to/directory/}}`

Reviewer: ```suggestion
`nitrogen {{path/to/directory}}`
```

- Set the wallpaper with automatic size settings:

`nitrogen --set-auto {{path/to/file}}`

- Restore the previous wallpaper:

> `nitrogen --restore &`

Reviewer: The `&` is to send the command to the background? it's not part of the app params itself.
```suggestion
`nitrogen --restore`
```

# nitrogen

> Nitrogen is a fast and lightweight desktop background browser and setter for X Window.
> More information: <https://github.com/l3ib/nitrogen/>.

- View and set the desired wallpaper/s from a specific directory:

> `nitrogen {{/path/to/directory/}}`

Reviewer 2: ```suggestion
`nitrogen {{path/to/directory/}}`
```

We do not use the leading `/` here.

- Set the wallpaper with automatic size settings:

> `nitrogen --set-auto {{/path/to/file}}`

Reviewer 2: ```suggestion
`nitrogen --set-auto {{path/to/file}}`
```
